sys-cluster/glusterfs/glusterfs-11.0-r3.ebuild

```diff
@@ -0,0 +1,217 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit autotools elisp-common python-single-r1 tmpfiles systemd bash-completion-r1
+
+DESCRIPTION="GlusterFS is a powerful network/cluster filesystem"
+HOMEPAGE="https://www.gluster.org/ https://github.com/gluster/glusterfs/"
+SRC_URI="https://download.gluster.org/pub/gluster/${PN}/$(ver_cut 1)/${PV}/${P}.tar.gz"
+
+LICENSE="|| ( GPL-2 LGPL-3+ )"
+SLOT="0/${PV%%.*}"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+
+IUSE="debug emacs firewalld +fuse georeplication ipv6 +libtirpc lto rsyslog selinux static-libs tcmalloc test +uring xml"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	georeplication? ( xml )
+	ipv6? ( libtirpc )"
+
+# the tests must be run as root
+RESTRICT="test"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-11.0-fix-crash.patch"
+	"${FILESDIR}/${PN}-11.0-extras-defer-invoking-of-gluster-volume-set-help-as-.patch"
+)
+
+# sys-apps/util-linux is required for libuuid
+RDEPEND="
+	acct-group/gluster
+	acct-user/gluster
+	dev-libs/libaio
+	dev-libs/openssl:=[-bindist(-)]
+	net-libs/rpcsvc-proto
+	dev-libs/userspace-rcu:=
+	sys-apps/util-linux
+	sys-libs/readline:=
+	${PYTHON_DEPS}
+
+	!elibc_glibc? ( sys-libs/argp-standalone )
+	emacs? ( >=app-editors/emacs-23.1:* )
+	firewalld? ( net-firewall/firewalld )
+	fuse? ( sys-fs/fuse )
```

Comment:
This isn't right. If it can be satisfied by either slot, it should be :*, but that's not right either unless it uses `dlopen()` and doesn't use headers/link against the relevant fuse.

We're also at the point where we really want to hard-choose fuse3 if we have a choice as fuse2 is deprecated.

How is it using fuse?